earth_enterprise/docker/README.md

```diff
@@ -169,7 +181,62 @@ image:
 ```
 
 You can optionally set the `OUTPUT_IMAGE_NAME` shell variable to override the
-default name used for the final Docker image.
+default name used for the final Docker image.  You can use the
+`OS_DISTRIBUTION` variable to set the platform you want to build on, the
+`STAGE_1_NAME` to set whether to build from a clean clone of the GEE
+repository, or from you current clone, and `CLEAN_CLONE_BRANCH` to build from
+the most recent commit on a given Git branch when building a clean clone.
+
+### Examples:
+
+Build from a clean repsitory clone on Cent OS 7:
+
+```BASH
+OS_DISTRIBUTION=centos-7 ./bin/build-docker-image.sh
+```
+
+
+Build from your current clone of the repository on Ubuntu 16:
+
+```BASH
+OS_DISTRIBUTION=ubuntu-16 STAGE_1_NAME=current-clone ./bin/build-docker-image.sh
+```
+
+
+Build a clean clone of branch `release_5.2.0` on Cent OS 7:
+
+```BASH
+OS_DISTRIBUTION=centos-7 CLEAN_CLONE_BRANCH=release_5.2.0 ./bin/build-docker-image.sh
+```
+
```

Comment:
No, they're for being able to tell apart the examples from each other easier when editing the source, rather than when looking at the rendered document.